pkg/common/ac_operations.go

```diff
@@ -278,8 +279,17 @@ func (a *ACOperationsImpl) tryToFindAC(acs []*accrd.AvailableCapacity, storageCl
 	var (
 		allocatedCapacity int64 = math.MaxInt64
 		foundAC           *accrd.AvailableCapacity
+		driveUUID         = make([]string, 0)
 	)
 	for _, ac := range acs {
+		// Available capacity with system disk location won't be allocated
+		if len(driveUUID) == 0 {
+			driveUUID = a.k8sClient.GetSystemDriveUUID(context.Background())
+			driveUUID = append(driveUUID, base.SystemDriveAsLocation)
+		}
+		if util.IsDriveSystem(ac.Spec.Location, driveUUID) {
```

Comment:
This code looks weird. You just get UUID of a system drive and later check whether is system or not.

Reply:
Fixed